nvim/after/ftplugin/markdown.lua

```diff
@@ -1,30 +1,20 @@
-local M = {}
-
-local config = require("nvim-surround.config")
-
 require("nvim-surround").buffer_setup({
   surrounds = {
     ["e"] = {
       add = function()
-        local env = require("nvim-surround.config").get_input "Environment: "
+        local env = require("nvim-surround.config").get_input("Environment: ")
         return { { "\\begin{" .. env .. "}" }, { "\\end{" .. env .. "}" } }
       end,
     },
     ["b"] = {
       add = { "**", "**" },
-      find = function()
-          return M.get_selection({ motion = "a**" })
-      end,
+      find = "**.-**",
       delete = "^(**)().-(**)()$",
     },
     ["i"] = {
       add = { "*", "*" },
-      find = function()
-          return M.get_selection({ motion = "a*" })
```

Comment:
The `motion = "..."` key is supposed to represent the `...` Neovim motion. For example, the `a'` motion allows you to perform actions like `va'`, `ca'`, and `da'`. However, this is not the case for `a*`, i.e. `da*` is invalid syntax (since `a*` does not exist as a built-in Neovim motion). Thus we fallback on using a Lua pattern to find the selection from which we delete the left/right surrounding pair.